### ISSUE TITLE
"CSS updated" status doesn't go away after updating CSS

### ISSUE THREAD

**Issue by yandeu** [yandeu/five-server#21]
Hi, I found a bug: "CSS updated" status doesn't go away after updating CSS

The status won't go away and creates a large stripe spanning from the top to the center of the page.

I'm using version 0.0.32 of five server and version 1.59 of visual studio code

_Originally posted by @LeonardoCastDev in https://github.com/yandeu/five-server/issues/1#issuecomment-893958533_

**Comment by yandeu** [comment]
Hi @LeonardoCastDev,

Can you please tell me more about what you see?

Five Server appends a DIV with the id "fiveserver-info-wrapper" to the body for displaying the "CSS updated".

Maybe your HTML or CSS conflicts with it?

Do you have an idea for a better implementation of "fiveserver-info-wrapper"? 

Thanks!

**Comment by LeonardoCastDev** [comment]
Yes, so when I update my CSS I get the [CSS updated banner](https://imgur.com/a/VmSHJoa) but it doesn't go away. [It also displays like this](https://imgur.com/a/VAXZPV4). But in some projects I get the "CSS updated" and it goes away and then when I update it again the banner doesn't show at all. I'm not sure if I'm the only one and if it is because of the version but I've also noticed that when I use the background property with locally storaged images I usually get those bugs. 

About the better implementation I'm not really sure as I just started learning programming.

Also, sorry If I can't explain myself properly, I'm still learning English, but I'll keep trying to see what is causing the problem with the  "CSS updated" banner.
